src/test/java/com/o19s/es/ltr/query/LtrQueryTests.java

```diff
@@ -301,10 +325,9 @@ public void checkModelWithFeatures(List<PrebuiltFeature> features, int[] modelFe
         String modelAsStr = ranker.model();
         RankerFactory rankerFactory = new RankerFactory();
         Ranker rankerAgain = rankerFactory.loadRankerFromString(modelAsStr);
-        float[] scoresAgain = new float[] {(float)rankerAgain.eval(rl.get(0)), (float)rankerAgain.eval(rl.get(1)),
-                (float)rankerAgain.eval(rl.get(2)), (float)rankerAgain.eval(rl.get(3))};
+        float[] scoresAgain = {(float)ranker.eval(rl.get(0)), (float)ranker.eval(rl.get(1)),
```

Comment:
@softwaredoug It's been a while since you made this change, but do you recall why you're referencing `ranker` here instead of `rankerAgain`?